PDXostc/common-app
# Changes to support new AMB version 13 set API.

Please code review back to commit ffe5a6444a79bc7a1c0749b7a6f6c1454866fffb; my main changes are in commit 65a6eaddb529f11087db7f7128aba5661f712c91

Test via:
1) Pull and build this version of common-app
2) Pull and build Dashboard.
3) Run dashboard on your local system and verify that AMB error messages:
“AMB: There was an error on the speed get.”, “could not find property VehicleSpeed”, “Uncaught TypeError: Cannot read property 'get' of undefined"
stop after ~10 repetitions.

NOTE: Top and bottom bars are missing. This is not a regression caused by code in this commit.

This commit is in preparation for a subsequent HVAC commit that will add navigator.vehicle.set functionality.
